locale/fi/LC_MESSAGES/django.po

```diff
@@ -3697,6 +3697,14 @@ msgstr ""
 msgid "Forget automatic redirection"
 msgstr "Unohda automaattinen uudelleenohjaus."
 
-#: userprofile/views.py:78
+#: userprofile/views.py:126
 msgid "No privacy notice. Please notify administration!"
 msgstr "Ei tietosuojailmoitusta. Ota yhteys ylläpitäjään!"
+
+#: userprofile/views.py:153
+msgid "No local accessibility statement. Please notify the site's owner!"
+msgstr ""
```

Comment:
"No local accessibility statement. Please notify the site's owner!"
"Ei paikallista saavutettavuusselostetta. Ilmoitathan tästä sivuston ylläpitäjälle!"